tgui/packages/tgui/interfaces/PreferencesMenu/MainPage.tsx

```diff
@@ -342,7 +342,11 @@ const createSetRandomization =
 
 const sortPreferences = sortBy<[string, unknown]>(([featureId, _]) => {
   const feature = features[featureId];
-  return feature?.name;
+  if (feature?.sortingPrefix) {
+    return feature.sortingPrefix + feature.name;
```

Comment:
appending sortingprefix instead of just using it can help avoid situations where 2 features have identical prefixes, since itll be almost always suffixed with something different